db/seed.sql

```diff
@@ -3165,6 +3165,7 @@ CREATE TABLE `checkout_agreement` (
 
 LOCK TABLES `checkout_agreement` WRITE;
 /*!40000 ALTER TABLE `checkout_agreement` DISABLE KEYS */;
+INSERT INTO `checkout_agreement` VALUES (1,'Terms and Conditions','By proceeding with this purchase, you agree to accept the terms and conditions as described on our website: \r\n\r\n<a href=\"https://www.histwest.org.au/terms-and-conditions\" target=\"_blank\">Read full terms and conditions</a>.\r\n',NULL,'I agree to the terms and conditions',1,1,1);
```

Comment:
I've made this essentially a link to the website, so that the content isn't duplicated, and also so that it makes more sense, because the content of the t&c we have, doesn't make sense in the context of a checkout.  This makes it clear that they are reusing the t&c from the website, which hopefully makes a bit more sense to the user.  However the client can change the text at any time after launch; this at least gives them the basic setup of how the t&c are enforced.

The last field, `mode`, is `1` ("Manual" in the UI) to make it a checkbox.  If it is `0` ("Automatic") then the t&c are implicitly accepted by continuing with the purchase, which is dubiously enforceable imo.